### PR TITLE
Added base64 encoding to messages

### PR DESCRIPTION
**Pull request by varolbora5**
Added base64 encoding and decoding on sending and receiving ends.
Not sure if it will work
Hope it does

### CODE REVIEW

**Comment by t0xic0der** [comment]
Could you please elaborate on the reason behind the changes made in `lien.py`? Thank you.

File: lien.py [hunk 1]
```diff
@@ -34,8 +34,9 @@ async def chatroom(websocket, path):
     await register(websocket)
     try:
         async for message in websocket:
-            print("< [" + str(time.ctime()) + "] " + str(message))
-            await notify_mesej(message)
+            decmesg = base64.b64decode(message)
```
[the remark this answers is not in the document]
Comment (t0xic0der) [reply lien.py:37]: The thing is, `lien.py` is not the server. It is `main.py`.

[the remark this answers is not in the document]
Comment (varolbora5) [reply lien.py:37]: I know, I encoded and decoded them on sending and receiving ends only

[the remark this answers is not in the document]
Comment (t0xic0der) [reply lien.py:37]: `main.py` is the server. `cnew.py` is the client. You have made changes to an outdated code.

**Pull request closed without merging**